src/main/java/Duke.java

```diff
@@ -1,10 +1,424 @@
+            if(extractedCommand == null || extractedCommand.trim().length() == 0){
+                throw new EmptyDescriptionException();
+            }
+
+            String at = command.substring(positionOfAt + 4, command.length());
+            if(at == null || at.trim().length() == 0){
+                throw new NoDateGivenException();
+            }
+
+            Event newEvent = new Event(extractedCommand, at);
+            tasksToDo.add(newEvent);
+            acknowledgeTaskAddition(newEvent);
+        } catch(NoDateGivenException e){
+            System.out.println();
+            System.out.println("You need to tell me when this event is at, master");
+        } catch(EmptyDescriptionException e){
+            System.out.println();
+            System.out.println("Sorry, but you need to describe what event this is, master");
+        }
+    }
+
+    /**
+    * Acknowledges the addition of task by printing
+    * that the bot understood the task
+    *
+    * @param task the current task (To Do, Deadline or Event)
+    */
+    private static void acknowledgeTaskAddition(Task task){
+        System.out.println(SEPARATING_LINE);
+        System.out.println("    Understood. I've added this into the list.. ");
+        printTaskStatus(task);
+        System.out.println(SEPARATING_LINE);
+        System.out.println();
+    }
+
+    /**
+    * Deletes the task that the user specifies
+    *
+    * @param command the current command passed from user input
+    */
+    public static void deleteTask(String command){
+        int indexTaskDelete = Integer.parseInt(command.substring(DELETE_LEN, command.length()));
+        Task currentTask = tasksToDo.get(indexTaskDelete-1);
+        System.out.println(SEPARATING_LINE);
+        System.out.println("Noted. I have deleted this task:");
+        printTaskStatus(currentTask);
+        tasksToDo.remove(indexTaskDelete-1);
+        System.out.println(SEPARATING_LINE);
+        System.out.println();
+    }
+
+    /**
+    * Prints the task status at hand
+    *
+    * @param task the current task (To Do, Deadline or Event)
+    */
+    public static void printTaskStatus(Task task){
+        System.out.println(task.toString());
+    }
+
+    /**
+    * Reads the saved task file and creates a
+    * file in the directory if it does not exist
+    */
+    private static void readTaskFile() throws IOException{
+        try{
+            Files.createDirectories(Paths.get(TASK_FOLDER));
+            Files.createFile(Paths.get(TASK_FILE));
+        } catch(FileAlreadyExistsException e){
+            // We do not need to add new file
+        }
+        
+        File f = new File(TASK_FILE);
+
+        Scanner fileInput = new Scanner(f);
+        while (fileInput.hasNext()) {
+            String line = fileInput.nextLine();
+
+            char taskType = line.charAt(0);
+            char taskStatus = line.charAt(2);
+            String taskDescription = line.substring(4);
+
+            if (taskType == 'T'){
```

Comment:
You may want to include spacing between ")" and "{" to abide by coding standards